packages/rocketchat-livestream/client/views/liveStreamTab.js

```diff
@@ -0,0 +1,156 @@
+/* globals popout */
+import toastr from 'toastr';
+
+function parseUrl(url) {
```

Comment:
A function comment could be helpful with an example url?

Possible alternate name: optionsFromUrl()